editor/gui/scene_tree_editor.cpp

```diff
@@ -369,26 +369,26 @@ void SceneTreeEditor::_add_nodes(Node *p_node, TreeItem *p_parent) {
 					msg_temp += String::utf8("•  ") + String(E.name) + "\n";
 				}
 			}
-		}
-		if (num_connections >= 1 || num_groups >= 1) {
-			if (num_groups < 1) {
-				msg_temp += "\n";
-			}
-			msg_temp += TTR("Click to show signals dock.");
+		} else {
+			msg_temp += "\n";
 		}
 
 		Ref<Texture2D> icon_temp;
 		SceneTreeEditorButton signal_temp = BUTTON_SIGNALS;
+		auto msg_temp_ = TTR("Click to show signals dock.");
```

Comment:
```suggestion
		String msg_temp_end = TTR("Click to show signals dock.");
```
`auto` is [not allowed](https://docs.godotengine.org/en/stable/contributing/development/cpp_usage_guidelines.html#auto-keyword), also needs a better name